src/ts/voiceDim.ts

```diff
@@ -135,6 +135,15 @@ const otherQueries = {
   'deep site': 'is:deepsight',
   wishlist: 'is:wishlist',
   wishlisted: 'is:wishlist',
+  favorite: 'tag:favorite',
+  keeper: 'tag:keep',
+  junk: 'tag:junk',
+  infusion: 'tag:infuse',
+  archived: 'tag:archive',
+  tagged: 'is:tagged',
+  'not tagged': '-is:tagged',
+  'with notes': 'is:hasnotes',
+  'without notes': '-is:hasnotes',
```

Comment:
This might actually be ok since the `split` looks for " with " (with spaces). I'd have to test it.

Reply:
try it and let me know. I do not have a microphone at my current location.